x-pack/osquerybeat/beater/config_plugin_test.go

```diff
@@ -11,9 +11,10 @@ import (
 	"path/filepath"
 	"testing"
 
-	"github.com/elastic/beats/v7/libbeat/logp"
 	"github.com/google/go-cmp/cmp"
 
+	"github.com/elastic/beats/v7/libbeat/logp"
+
```

Comment:
nit: should there be an empty line?

Reply:
This is what `mage ftm` gave, but yes removing is ok. Thanks!